src/components/InformationsBar/CloroplethLegend/CloroplethLegend.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import 'jest-styled-components';
+import { render, screen } from '@testing-library/react';
+import { CloroplethLegend } from '.';
+import { Container, Title, Legend, GradientBar } from './styles';
```

Comment:
```suggestion
import 'jest-styled-components';

import React from 'react';

import { render, screen } from '@testing-library/react';

import { CloroplethLegend } from '.';
import { Container, Title, Legend, GradientBar } from './styles';
```

Trying to have a more clear header in ours component can improve the file readability as well, WDT?